test/Microsoft.DotNet.GenAPI.Tests/CSharpFileBuilderTests.cs

```diff
@@ -2149,7 +2149,7 @@ public partial interface IntType
                  includeInternalSymbols: false);
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/67019")]
+        [Fact]
```

Comment:
Unskipping failed. Could not reopen original issue. New issue with details here: https://github.com/dotnet/roslyn/issues/74109